resources/todomvc/architecture-examples/svelte/README.md

```diff
@@ -0,0 +1,36 @@
+# Speedometer 3.0: TodoMVC: Svelte
```

Comment:
it could be good to mention that this has been heavily inspired by https://github.com/sveltejs/svelte-todomvc